traits/has_traits.py

```diff
@@ -1358,8 +1358,14 @@ def trait_get(self, *names, **metadata):
 
         Parameters
         ----------
-        names : list of strings
-            A list of trait attribute names whose values are requested.
+        names : optional
+            List of positional arguments. Either trait attribute names as
+            string arguments or a single list or tuple of trait attribute
+            names whose values are requested.
+        metadata : optional
+            Dictionary of keyword–value pairs. Metadata information used to
```

Comment:
Suggest dropping the "Dictionary of keyword-value pairs" part here: with the suggested change above, we already know it's a dictionary, and "keyword-value pairs" doesn't add any information (it describes _all_ dictionaries :-).